functional.py

```diff
@@ -0,0 +1,44 @@
+import sys
+#Task 1
+def sequential_map(*args):
+    # Unpack all the arguments, write the functions to a list, and write the last argument as a container.
+    *func_lst, conteiner = [*args]
+    for func in func_lst:
+        # Overwrite to the container the result of the function applied to each element.
+        conteiner = func(conteiner)
+    return [x for x in conteiner]
```

Comment:
Функции должны применяться к элементам контейнера, а не к нему самому
```suggestion
    for func in func_lst:
        # Overwrite to the container the result of the function applied to each element.
        conteiner = map(func, conteiner)
    return list(conteiner)
```